TypeCobol/Compiler/Scanner/Scanner.cs

```diff
@@ -2291,7 +2301,7 @@ private Token ScanUntilDelimiter(int startIndex, TokenType tokenType, string del
         /// <summary>
         /// Look for pattern ':' (cobol word chars)+ ':'
         /// </summary>
-        private bool CheckForPartialCobolWordPattern(int startIndex, out int patternEndIndex)
+        private bool CheckForPartialCobolWordPattern(MultilineScanState currentState, int startIndex, out int patternEndIndex)
```

Comment:
the state can be read directly by the method using the instance field `tokensLine` so the new parameter is not required.

Reply:
ok

Reply:
done